src/components/NewMessageModal.vue

```diff
@@ -137,9 +136,9 @@ export default {
 				})
 			}
 
-			if (!data.sendAt || data.sendAt < Math.floor((now + UNDO_DELAY) / 1000)) {
+			if (!data.sendAt || data.sendAt < Math.floor((now + TOAST_UNDO_TIMEOUT) / 1000)) {
 				// Awaiting here would keep the modal open for a long time and thus block the user
-				this.$store.dispatch('outbox/sendMessageWithUndo', { id: this.composerData.id })
+				await this.$store.dispatch('outbox/sendMessageWithUndo', { id: this.composerData.id })
```

Comment:
Are you sure to await here? See the comment one line above.

Reply:
i dont remember adding that, maybe phpstorm lint did that?